extensions/amp-access/0.1/test/test-amp-access.js

```diff
@@ -1474,7 +1473,7 @@ describes.fakeWin('AccessService analytics', {
 });
 
 
-describes.fakeWin('AccessService multiple sources', {
+describes.realWin('AccessService multiple sources', {
```

Comment:
Why the changes in this file?

Reply:
amp-access tests were failing because performance is now is not present in a fakeWin